buildtest/cli/config.py

```diff
@@ -22,6 +22,9 @@ def config_cmd(args, configuration, editor, system):
     if args.config in ["view", "v"]:
         view_configuration(configuration, theme=args.theme, pager=args.pager)
 
+    if args.config in ["profile", "v"]:
```

Comment:
You can see here that  "v" is already used in the above if statement, this could cause conflicts. Aside from that, I think it looks fine, we'll have to see once we run the code

Reply:
You could try adding "p" so that there still is a shorter option than `profile`

Reply:
cant, it will conflict wit line 48